server/controllers/bookController.js

```diff
@@ -74,6 +74,49 @@ class bookController {
         error: error.message,
       });
     }
+  };
+  static async deleteBook(req, res){
+
+    try{
+       const bookId = req.params.bookId;
+       if(isNaN(bookId)){
+        return res.status(400).json({
+          status: 400,
+          error: 'Book id must be integer',
+        });
+       }
+       const bookExist = await Books.findOne({
+        where: {
+          id: bookId.trim(),
+        },
+      });
+
+      if (!bookExist) {
+        return res.status(404).json({
+          status: 404,
+          error: 'Book is not found',
+        });
+      }
+     const deletedBook = await Books.destroy({
+          where:{
+            id: bookId
+          }
+     })
+     if(deletedBook){
+       return res.status(200).json({
+        status: 200,
+        message: 'Book deleted successfully',
+        data: []
+      })
```

Comment:
i this it would have been better if you returned 
status 204
`A 204 ( No Content ) status code if the action has been enacted and no further information is to be supplied.`

and you don't need to return _data:[ ]_

Reply:
@blaise82 Thank you, I get you but when you want to return back a message to admin after deleting a book, it can't be possible using ``` 204 `` status code